tests/test_tf.py

```diff
@@ -554,29 +554,31 @@ def circuit_tf(phi, theta):
 ]
 
 
-dev = qml.device("default.qubit", wires=2)
-
-
-@qml.qnode(dev, interface="tf")
-def f(x):
-    qml.RX(x, wires=0)
-    return qml.expval(qml.PauliZ(0))
-
+@pytest.mark.usefixtures("skip_if_no_tf_support")
```

Comment:
(A question unrelated to this specific change)

Is there a certain reason why ``tf_support`` is defined in this file (lines 46-56), whereas ``torch_support`` is defined in ``conftest.py``?
```python

@pytest.fixture(scope='module')
def tf_support():
    """Boolean fixture for TensorFlow support"""
    try:
        import tensorflow as tf
        tf_support = True

    except ImportError as e:
        tf_support = False

    return tf_support
```

Reply:
Also, I am wondering about the ``scope`` for these two functions, is there a reason why for ``tf_support`` it's set to ``'module'``, but for ``torch_support`` it's set to ``'session'``?

Reply:
These changes have been made in #357, so I won't touch them here to avoid merge conflicts :)